### ISSUE TITLE
Summary table misaligned when digits_summary large enough

### ISSUE THREAD

**Issue by jjramsey** [stan-dev/pystan#429]
The text table produced by `stansummary` can be non-readable if digits_summary is large enough.

Here's an example with the default digits_summary:

```
           mean se_mean     sd   2.5%    25%    50%    75%  97.5%  n_eff   Rhat
mu         7.96    0.16   5.15  -2.25   4.79   7.79  11.14  18.39   1088    1.0
theta[0]  11.55    0.26   8.35  -1.89   6.03  10.21  15.96  31.95   1009    1.0
theta[1]   7.98    0.14   6.41  -4.81   4.19   7.84  11.94  21.39   1997    1.0
theta[2]   6.19    0.19   7.82 -11.77   2.32   6.56  10.66  21.38   1721   1.01
theta[3]   7.61    0.17   6.61  -5.93   3.69   7.46  11.62  21.27   1601    1.0
theta[4]   5.01    0.19   6.22  -8.71   1.23   5.55   9.14  16.22   1061   1.01
theta[5]   5.89    0.16   6.64  -9.14    2.2   6.28  10.15   18.3   1647    1.0
theta[6]  10.84     0.2   6.68  -0.59   6.23  10.04  14.69  25.94   1127    1.0
theta[7]   8.62     0.2   8.19  -7.96   4.05   8.26  12.98  26.64   1716    1.0
tau        6.94    0.32    5.6   0.78   2.92    5.7   9.36  20.33    302   1.01
lp__     -17.62    0.48    5.9 -27.75 -21.64 -18.44 -14.11  -4.25    154   1.02
```

Here's an example where digits_summary = 5:

```
           mean se_mean     sd   2.5%    25%    50%    75%  97.5%  n_eff   Rhat
mu       7.9551  0.1560 5.1468 -2.254 4.7893 7.78741.114e11.8388e1   1088 1.0035
theta[0]1.1547e1  0.2629 8.3521 -1.885 6.02781.0209e11.5956e13.1949e1   1009 1.0013
theta[1] 7.9804  0.1435 6.4125 -4.812 4.1925 7.83551.1939e12.139e1   1997 1.0020
theta[2]  6.194  0.1886 7.8245-1.1765e1 2.3196 6.55821.0662e12.1375e1   1721 1.0059
theta[3] 7.6119  0.1651 6.6068 -5.930 3.6864 7.46181.1616e12.127e1   1601 1.0025
theta[4] 5.0118  0.1908 6.2172 -8.714 1.2324 5.5505 9.13911.6219e1   1061 1.0067
theta[5] 5.8893  0.1637 6.6442 -9.137 2.2025 6.27571.0146e11.8301e1   1647 1.0027
theta[6]1.0845e1  0.1988 6.6763 -0.592 6.23481.0045e11.4692e12.5943e1   1127 1.0008
theta[7] 8.6178  0.1978 8.1945 -7.956 4.0482 8.26091.2984e12.6635e1   1716 1.0004
tau      6.9445  0.3221 5.5985 0.7833 2.9173 5.6959 9.35592.0332e1    302 1.0107
lp__    -1.7616e1  0.4754 5.8998-2.7745e1-2.1645e1-1.8442e1-1.4109e1 -4.253    154 1.0213
```

Note that RStan does not have this problem.

**Comment by ahartikainen** [comment]
Thanks for the bug report.

Looks like a problem with scientific notation.

**Comment by ariddell** [comment]
The durable solution here is to have consistent formatting provided by Stan. Right now we have 3 different implementations (RStan, CmdStan, PyStan).

**Comment by mdastro** [comment]
Dear all, 

I have been dealing with the same issue for a long time (and procrastinating to report it). For practical reasons, I have manually fixing the summary output. Nonetheless, it would be highly beneficial to create an alternative output for the summary in order to avoid this issue (e.g. an array). 

I am  attaching an image with this problem.
![Screenshot from 2019-07-03 06-42-34](https://user-images.githubusercontent.com/10452764/60581768-e1f59c00-9d5d-11e9-8812-c45d5216ab1b.png)

Thank you for the attention! :-)

**Comment by ahartikainen** [comment]
Not a fix: have you tried `arviz.summary`?

I think I can next work to fix the output of summary function.

**Comment by ahartikainen** [comment]
I mean we need to update our summary (arviz has updated stats), and I just really would want to wait for stan-math version for the analysis.

**Comment by mdastro** [comment]
I have used `arviz` for a few things, such as for plotting. But I haven't used the summary function. I might try it indeed.

I hope the stan-math version you mentioned gets ready soon. I look forward for the summary function improvements. :-)

Thanks again for the prompt reply!

**Comment by ahartikainen** [comment]
Fixed by #618
